source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/Models/FeatureClassUtils.cs

```diff
@@ -361,18 +361,327 @@ private IFeatureClass ExportToShapefile(string fileNamePath, List<Graphic> graph
 
                     fieldEdit.GeometryDef_2 = geomDef;
                     fieldsEdit.AddField(field);
+                    string graphicsType = graphicsList[0].GraphicType.ToString();
+                    switch(graphicsType)
+                    {
+                        case "Line":
+                            {
+
+                                field = new FieldClass();
```

Comment:
Can this code be combined in a method with: https://github.com/Esri/distance-direction-addin-dotnet/blob/dev/source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/Models/FeatureClassUtils.cs#L481  ? They appear to be the same